Extension/src/LanguageServer/extension.ts

```diff
@@ -1028,10 +1038,10 @@ function handleCrashFileRead(err: NodeJS.ErrnoException, data: string): void {
     const process2: string = "Microsoft.VSCode.CPP.Extension.darwin\t";
     if (data.includes(process1)) {
         data = data.replace(new RegExp(process1, "g"), "");
-        data = process1 + "\n" + data;
+        data = `${process1}${binaryVersion}\n${data}`;
```

Comment:
do you want to separate your binary name with your version string with a `.` so its easier to parse?

Reply:
It's already separated by '\t'.